css/main.css

```diff
@@ -0,0 +1,500 @@
+/* general */
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  min-height: 100vh;
+  background-color: rgba(236, 236, 236, 0.952);
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+}
+
+main {
+  background-color: white;
+}
+
+/* nav */
+
+.nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+  background-color: white;
```

Comment:
- [ ] **MANDATORY** You fixed some of the reapeated classes from last time but not all, this background-color: white repeats 7 times in your CSS.
* For good coding practices and to follow the DRY (Don't repeat yourself) rule, you have 2 options, this will also help you when you have to do a change for all similar elements:
- use multiselection:
```
.a1 p, .a1 p, .a2 p, .a4 p {
   ... (Your styles)
}
```
* OR  you can use multiple classes for any element, so for example,

  If you use this line in many classes:
  `color: #fff;` and you use it for .class1, .class2, and .class3 
  you can put it inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px and color of #fff -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px and color of #fff -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px and color of #fff -->
  ...
  ```